pypusu/polling.py

```diff
@@ -1,9 +1,13 @@
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
 import json
 from .threaded import PuSuClient as ThreadedPuSuClient
 
 try:
     # Py2
-    from Queue import Queue, Empty
```

Comment:
Sure seems like you're breaking Python 2 support with this, are you sure this is what you meant to do?

Reply:
Ah, this might be a consequence of the aliasing.

Reply:
So I should fix it back to 'from Queue import Queue, Empty' right?

Reply:
No, what you should do is make sure this change is correct.

I *believe* 

```python
from future import standard_library
standard_library.install_aliases()
```

... makes the `from queue` work instead of the `from Queue` which it seems to have been in Python 2. If that is the case, and both Python 2 and Python 3 imports are the same, then the `try: ... except ImportError: ...` should be replaced with just the plain `import` that works on both.

It's up to you to confirm that this code works correctly on both Python 2.7 and 3.6+

Reply:
<img width="822" alt="Screenshot 2020-03-04 at 11 24 59" src="https://user-images.githubusercontent.com/23077311/75864785-39776a80-5e0b-11ea-86c9-bc60085024dc.png">

So I checked, apparently `from queue` works also for Python 2

Reply:
When you say you checked, you mean that you actually tested it on Python 2 and Python 3, and did not just look at code?

Reply:
I tested by importing the queue module in Python 2.7. Then i checked the code to see what is imported. And in the file it imports everything from the Queue module.

Reply:
So you're saying you've not tested these actual changes?

Reply:
Sorry for not testing it carefully before. Now I have tested. It works fine both on Python 2.7 and Python 3.6.